bin/add_attachment_revision.py

```diff
@@ -0,0 +1,181 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2021 Xinru Yan <xinru1414@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Used to add attachment revision as an additional attachament to the Anthology.
+
+This script is heavily adopted from add_attachments.py
+Only adding attachment without reading in .csv file and keeping a log
+
+Usage:
+
+    add_attachment_revision.py anth_id attachment_revision_URL 'ATTACHMENT_TYPE'
+"""
+
```

Comment:
Can you do the following script renaming?

* add_attachment.py → add_attachment_bulk.py
* add_attachment_revision.py → add_attachment.py